docs/index.rst

```diff
@@ -4,9 +4,10 @@
 build
 *****
 
-A simple, correct :pep:`517` build frontend.
+A simple, correct Python build frontend.
```

Comment:
```suggestion
A simple, correct Python packaging build frontend.
```

This still feels a bit incomplete to me, but it's better.

Reply:
This PR matches what the README has. :)